config/locales/se.yml

```diff
@@ -0,0 +1,14 @@
+se: 
+  flash[:error]: Fält kan inte vara tomma. Din artikel kunde inte sparas
+  article_successfully_created: Artikel skapad.
+  published_on: Publicerad 
+  Registration: Registrering
+  password: Lösenord
+  characters_minimum: minimum tecken
+  new-password: Nytt lösenord
+  remember_me: Kommer ihåg mig
+  Sign_up: Signa upp mig
```

Comment:
sign_up so everything is with small letters.
We need to change Sign_up to sign_up in the views though